### ISSUE TITLE
Migrate from Moralis to `The Graph`

### ISSUE THREAD

**Issue by advaita-saha** [metafy-social/eth-unwrapped#10]
### Is your feature request related to a problem? Please describe
Currently moralis is integrated for retrieving blokchain data, which is inherently slow for api calls.
Migrate all the calls from moralis to `the graph` for faster and better blockchain data

**Comment by SakshiShah29** [comment]
I would like to work on this issue. Please assign it to me under GSSoC'23.

**Comment by advaita-saha** [comment]
@SakshiShah29 connect with me or @shahkushal38 over discord/linkedin
If you are working on it, you will need help as this would be a major codebase revamp

Wish you best of luck

**Comment by SakshiShah29** [comment]
yaa sure 
My discord is GirlGonoph💜#0483

**Comment by shahkushal38** [comment]
Yes, you can connect on the eth-unwrapped discord channel where Advaita and me can help you. My discord is kushal#5520
